polygon_cli/actions/diff.py

```diff
@@ -3,9 +3,10 @@
 
 
 def process_diff(options):
+    # resolve paths before load_session changes working directory
     input_files = list(map(os.path.abspath, options.file))
 
-    if not load_session():
+    if not load_session(verbose=options.verbose):
```

Comment:
I support reducing copy paste. But passing options structure directly to `process_action` doesn't seem right either. It would make testing or reusing `process_action` functions difficult. What do you think expanding those command line option handling lamdas like this:

```python
def process_action(action_arg1, action_arg2):
      # main action logic here

def handle_action_options(options):
     if not load_session_from_opts(options)
         fatal("error")
     process_action(options.action_arg1, options.action_arg2)
     save_session()
```
They could be merge in situations where process_action is trivial like`process_download_last_package`